File: app/src/main/java/com/vuphan/walking__m_hike_2/bottomnav/HomeFragment.java
package com.vuphan.walking__m_hike_2.bottomnav;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ImageView;

import androidx.fragment.app.Fragment;

import com.vuphan.walking__m_hike_2.R;


public class HomeFragment extends Fragment {

    ImageView iconHeart1, iconHeart2;
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @SuppressLint("MissingInflatedId")
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {

        View view = inflater.inflate(R.layout.fragment_home, container, false);
//        iconHeart1 = view.findViewById(R.id.icon_heart);
//        iconHeart2 = view.findViewById(R.id.icon_heart2);
        return view;
    }

    @Override
    public void onResume() {
        super.onResume();
//        iconHeart1.setOnClickListener(view -> {
//
//        });
    }
}

File: app/src/main/java/com/animsh/runningtracker/bottomnav/HomeFragment.kt
package com.animsh.runningtracker.bottomnav

import android.os.Bundle
import android.view.View
import androidx.fragment.app.Fragment
import androidx.fragment.app.viewModels
import androidx.lifecycle.Observer
import androidx.navigation.fragment.findNavController
import androidx.recyclerview.widget.LinearLayoutManager
import com.animsh.runningtracker.R
import com.animsh.runningtracker.adapters.PostAdapter
import com.animsh.runningtracker.ui.viewmodels.PostViewModel
import dagger.hilt.android.AndroidEntryPoint
import kotlinx.android.synthetic.main.fragment_home.*

@AndroidEntryPoint
class HomeFragment : Fragment(R.layout.fragment_home) {

    private val viewModel: PostViewModel by viewModels()
    private lateinit var postAdapter: PostAdapter

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
//        setupRecyclerView()

        imageView.setOnClickListener{
            findNavController().navigate(R.id.action_menu_home_to_postFragment)
        }
//
//        viewModel.posts.observe(viewLifecycleOwner, Observer {
//            postAdapter.submitList(it)
//        })
    }

//    private fun setupRecyclerView() = rvPost.apply {
//        postAdapter = PostAdapter()
//        adapter = postAdapter
//        layoutManager = LinearLayoutManager(requireContext())
//    }
}